lib/compat/wordpress-6.3/class-gutenberg-rest-global-styles-controller-6-3.php

```diff
@@ -18,6 +18,47 @@ class Gutenberg_REST_Global_Styles_Controller_6_3 extends Gutenberg_REST_Global_
 	 */
 	private $revisions_controller;
 
+	/**
+	 * Registers the controllers routes.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+		parent::register_routes();
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_site_item' ),
+					'permission_callback' => array( $this, 'get_theme_items_permissions_check' ),
+				),
+			)
+		);
+
+	}
+
+	/**
+	 * Return the global styles config for the site origin.
+	 *
+	 * @since 6.3.0
+	 *
+	 * @param WP_REST_Request $request The request instance.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function get_site_item( $request ) {
```

Comment:
Updated in https://github.com/WordPress/gutenberg/pull/50102/commits/9e7c951d4851cd3eacfa3e43a6ece7fcfaea3f5b to follow the existing patterns in [get_item](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php#L210) (error management, etc.).

Reply:
cool, thanks